### PR TITLE
Feature/add profiles api

### PR DESCRIPTION
**Pull request by EugeneTryF**
created new Profile API with standart CRUD, entities - Account, Patient, Receptionist, Doctor, Specialization, this entities will be removed from Clinic API after full db deconstraction on different APIs

### CODE REVIEW

File: common/StandartCRUD/StandartBLL/IGenericService.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿namespace Clinic.BLL.Interfaces
+﻿namespace StandartCRUD.StandartBLL
 {
     public interface IGenericService<TEntity, TModel>
```
Comment (VadimZharikov) [inline common/StandartCRUD/StandartBLL/IGenericService.cs]: redundant entry TEntity

File: src/Profiles/Profiles.API/Controllers/AccountController.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AccountController(IAccountService accountService, IMapper mapper) : ControllerBase
+    {
+        private readonly IAccountService _accountService = accountService;
+        private readonly IMapper _mapper = mapper;
+
+        [HttpGet]
+        public async Task<IEnumerable<AccountViewModel>> GetAll(string? email, string? phoneNumber, bool? isActive, CancellationToken cancellationToken)
```
Comment (VadimZharikov) [inline src/Profiles/Profiles.API/Controllers/AccountController.cs]: could also be externalised to ViewModel/record

File: common/StandartCRUD/StandartBLL/IGenericService.cs (hunk 1 above)
Comment (EugeneTryF) [reply common/StandartCRUD/StandartBLL/IGenericService.cs]: TEntity removed

File: src/Profiles/Profiles.API/Controllers/AccountController.cs (hunk 2 above)
Comment (EugeneTryF) [reply src/Profiles/Profiles.API/Controllers/AccountController.cs]: done

**Review by VadimZharikov — APPROVED** [review]

**Review by Genresal — APPROVED** [review]
LGTM

**Pull request merged**